src/jwt.cc

```diff
@@ -64,6 +64,7 @@ Status Jwt::parseFromString(const std::string& jwt) {
   if (alg_ != "ES256" && alg_ != "ES384" && alg_ != "ES512" &&
       alg_ != "HS256" && alg_ != "HS384" && alg_ != "HS512" &&
       alg_ != "RS256" && alg_ != "RS384" && alg_ != "RS512" &&
+      alg_ != "PS256" && alg_ != "PS384" && alg_ != "PS512" &&
```

Comment:
can we define a static absl::flat_hash_set to to optimize this checking. 

Please define it inside an anonymous namespace for the static set.

Reply:
Added in commit 9119883!